crypto/ml_kem/ml_kem_768.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* Copyright (c) 2024, Google Inc. */
```

Comment:
Whatever we decide to do with attributing for google, it probably needs to be consistent across files.